content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,74 @@ menu:
 weight: 101
 ---
 
+## v2.0.0-rc.0 [2020-09-29]
+
+{{% warn %}}
+#### Manual upgrade required
+
+To simplify the migration for existing users of InfluxDB 1.x, this release includes significant breaking changes that require a manual upgrade from all alpha and beta versions. For more information, see [Upgrade to InfluxDB OSS 2.0rc0](/influxdb/v2.0/reference/rc0-upgrade-guide/),
+{{% /warn %}}
+
+### Breaking changes
+
+#### Manual upgrade
+
+- To continue using InfluxDB 2.0 beta 16 or earlier, you must move all existing data out of the `~/.influxdbv2` (or equivalent) path, including `influxd.bolt`. All existing dashboards, tasks, integrations, alerts, users, and tokens must be recreated. For information on how to migrate your data, see [Upgrade to InfluxDB OSS 2.0rc0](/influxdb/v2.0/reference/rc0-upgrade-guide/).
```

Comment:
@russorat do we mean to continue using data from beta 16 or earlier?

Reply:
just the data. they can keep running beta 16 if they want.